### PR TITLE
revised PR#80, added new submissions schema and seed data

### PR DESCRIPTION
**Pull request by Aaron-TheCreator**
# Description
this PR creates a new table in the db to replace the current Submissions table
Fixes #80 

- What work was done?
- columns deleted from previous submissions table, new columns added in their place
- Why was this work done?
- booleans were removed, replaced with timestamps, some info will be stored in (yet to be created) 'Pages' table
- What feature / user story is it for?
- https://trello.com/c/J7brkjtW/464-schema-changes-to-the-submissions-table
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## Change Status

- [X] Complete, but not tested (may need new tests)

## Has This Been Tested

- [X] No

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

**Comment by Zerkue** [comment]
Great work! The comments explain the code fairly well and I can understand what is going on step by step.

File: data/seeds/019_Submissions-New.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+};
+
+//FYI, each submission number here represents a week that was completed by a number of students.
+//So, *groups 1,2&3 finished week1. 3 groups of 8 students = 24 items in this batch
+const submissions1 = [...new Array(24)].map((i, idx) => ({
+  childId: `${idx + 1}`,
+  storyId: 1,
+  episodeId: 1,
+  episodeStartDate: '2021-01-03',
+  // moderationStatus replaced Status in previous seed
```
Comment (chukalicious) [inline data/seeds/019_Submissions-New.js:15]: Thank you! 'Status' is not very descriptive

**Review by chukalicious — APPROVED** [review]
I actually like the way seeds are created here. Thank you

**Review by chukalicious — APPROVED** [review]

**Review by tonysorensen — APPROVED** [review]
Changes are in line with the new schema discussed. Appropriate seed data.

**Pull request merged**